scss/_grid.scss

```diff
@@ -23,10 +23,17 @@ Mobile First - This applies from 0px to 767px
 .row .row [class^="grid"]:only-child            { padding:0; }
 .row .row [class^="grid"]:last-child            { padding-right:0; }
 
-/***** Grid gutters *****/
-.grid, .grid-1, .grid-2, .grid-3, .grid-4,
-.grid-5, .grid-6, .grid-7, .grid-8, .grid-9,
-.grid-10, .grid-11, .grid-12                    { box-sizing:border-box; width:100%; }
+/***** Sets grid widths *****/
+.grid                                           { box-sizing:border-box; width:100%; }
+
+@for $i from 1 through $grid-columns            {
+  .grid-#{$i}                                   { box-sizing:border-box; width:100%; }
+}
+
+/***** Sets mobile override grid width *****/
+@for $i from 1 through $grid-columns {
+  .grid-#{$i}-mobile                            { flex-grow:$i; max-width:  $i / $grid-columns * 100%; }
```

Comment:
Shouldn't these be within a media query as they only activate on mobile.
Otherwise (mobile first) we have to undo the properties in sizes greater than mobile.
Ref: #88